i18n/zh/docusaurus-plugin-content-docs-community/current/5-release-guide.md

```diff
@@ -615,72 +536,67 @@ announcements in the coming days.
 
 ## 完成发布
 
-**1. 将源码、二进制包从svn的dev目录移动到release目录，并删除release目录的前一个版本**
-
-```shell
-svn mv https://dist.apache.org/repos/dist/dev/incubator/shenyu/${RELEASE.VERSION} https://dist.apache.org/repos/dist/release/incubator/shenyu/ -m "transfer packages for ${RELEASE.VERSION}"
-svn delete https://dist.apache.org/repos/dist/release/incubator/shenyu/${PREVIOUS.RELEASE.VERSION}
-```
-
-**2. 在Apache Staging仓库找到ShenYu并点击`Release`**
-
-**3. 合并Github的release分支到`master`, 合并完成后删除release分支**
+**1. 完成 SVN 发布**
 
-从GitHub Fork一份代码，并执行以下命令：
+根据 [Uploading packages](https://infra.apache.org/release-publishing.html#uploading) [6] 的说明将新版本从dev目录转移到release目录。
 
 ```shell
-git checkout master
-git merge origin/${RELEASE.VERSION}-release
-git pull
-git push origin master
+svn mv https://dist.apache.org/repos/dist/dev/incubator/shenyu/${PUBLISH.VERSION} https://dist.apache.org/repos/dist/release/incubator/shenyu/ -m "transfer packages for ${PUBLISH.VERSION}"
+svn delete https://dist.apache.org/repos/dist/release/incubator/shenyu/${PREVIOUS.RELEASE.VERSION}
 ```
 
-以上修改创建一个pull request。
-
-在项目原始仓库执行以下命令：
-
-```shell
-git push --delete origin ${RELEASE.VERSION}-release
-git branch -d ${RELEASE.VERSION}-release
-```
+**2. 完成 Maven 发布**
 
-**4. 发布 Docker**
+根据 [publishing maven artifacts](https://infra.apache.org/publishing-maven-artifacts.html) [4] 的说明完成 Maven 发布。
 
-4.1 准备工作
+回到 [Staging仓库](https://repository.apache.org/#stagingRepositories) 的 ${STAGING.RELEASE}，点击 `Release`。
 
-本地安装 Docker，并启动服务。
+**3. 完成 Docker 发布**
 
-4.2 编译 Docker 镜像
+安装 [Docker](https://docs.docker.com/get-docker/) 。
 
 ```shell
-git checkout v${RELEASE.VERSION}
+git checkout v${PUBLISH.VERSION}
 cd ~/shenyu/shenyu-dist/
 mvn clean package -Prelease,docker
-```
-
-4.3 发布 Docker 镜像
-
-```shell
 docker login
 docker push apache/shenyu-bootstrap:latest
 docker push apache/shenyu-bootstrap:${RELEASE_VERSION}
 docker push apache/shenyu-admin:latest
 docker push apache/shenyu-admin:${RELEASE_VERSION}
 ```
 
-4.4 确认发布成功
+登录 Docker Hub 验证 [shenyu-bootstrap](https://hub.docker.com/r/apache/shenyu-bootstrap/) 和 [shenyu-admin](https://hub.docker.com/r/apache/shenyu-admin/) 的镜像是否存在。
+
+**4. 完成 GitHub 发布**
+
+编辑 [Releases](https://github.com/apache/incubator-shenyu/releases) 中的 `${PUBLISH.VERSION}`，然后点击发布。
+ 
+**5. 完成 GitHub 更新**
 
-登录 Docker Hub 查看 [shenyu-bootstrap](https://hub.docker.com/r/apache/shenyu-bootstrap/) 和 [shenyu-admin](https://hub.docker.com/r/apache/shenyu-admin/) 是否有发布的镜像
+从 GitHub Fork 一份代码，并执行以下命令：
 
-**5. GitHub版本发布**
+```shell
+git checkout master
+git merge origin/${PUBLISH.VERSION}-release
+git pull
+git push origin master
+```
 
-在 [GitHub Releases](https://github.com/apache/incubator-shenyu/releases) 页面的 `${RELEASE_VERSION}` 版本上点击 `Edit`
+以上修改需要创建一个pull request。pr合并后，在原始仓库执行以下命令：
 
-编辑版本号及版本说明，并点击 `Publish release`
+```shell
+git push --delete origin ${PUBLISH.VERSION}-release
+git branch -d ${PUBLISH.VERSION}-release
+```
```

Comment:
update github release

Reply:
The content already in the `4. Finish GitHub release` section.